### PR TITLE
 Add sensors details in popup table

### PR DESCRIPTION
**Pull request by nerdlet**
## Description
Fixes #165414013

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/cell-info.vue [hunk 1]
```diff
@@ -9,48 +9,55 @@
 		p The numbers in the first column correspond to the sensor IDs. The first line 'mean' contains the respective average values  of all sensors contained in the cell.
 		p Please note: We show on the map the values of the last 5 minutes. The values published by the respective state authorities are given as a 24-hour average. This may cause the values on the chart to differ significantly from these 24-hour averages.
 		p By clicking on the plus in front of the sensor ID 2 graphics can be displayed. The '24 h floating 'graph displays the 24-hour moving average for the last 7 days. For technical reasons, there is a gap of one day at the beginning, so the presentation actually shows 8 days, but the first one is empty. The second graphic 'Last 24 hours' shows the course of the day for the last 24 hours.
-	h3 #Sensors {{cell.length}}
+	h3 Active sensors:  {{cell.length}}
```
Comment (karimkawambwa) [inline src/cell-info.vue]: They are not necessarily active
Its the number of sensors clumped together in that map pinpoint

File: src/feinstaub-api.js [hunk 2]
```diff
@@ -1,61 +1,54 @@
...
+          let id = function(id) {
+            for (let i in value.stats) {
+              return Number(value.stats[i].sensor_id);
+            }
+          };
+          let lat = Number(value.location.latitude);
+          let long = Number(value.location.longitude);
+          let location = String(value.location.name);
+          let cityName = String(value.location.city.name);
+          let date = Date(value.last_date_pushed);
```
Comment (karimkawambwa) [inline src/feinstaub-api.js]: This is now `last_data_received_at`

**Review by karimkawambwa — CHANGES_REQUESTED** [review]
Looks great 👍

File: src/cell-info.vue (hunk 1 above)
Comment (nerdlet) [reply src/cell-info.vue]: @karimkawambwa Working on it

**Review by karimkawambwa — APPROVED** [review]
LGTM

**Comment by karimkawambwa** [comment]
When ready to merge, remember to use production: `https://api.sensors.africa`

**Review by kilemensi — CHANGES_REQUESTED** [review]
Good stuff @nerdlet :+1: 

No need for `momentjs` though **if** all you're using it for is date format: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString is good enough.

**Comment by nerdlet** [comment]
@kilemensi @karimkawambwa Sensor ids are displaying now on production

**Comment by karimkawambwa** [comment]
Should we merge this and release ? @nerdlet

**Pull request merged**